scene/gui/code_edit.h

```diff
@@ -332,6 +334,9 @@ class CodeEdit : public TextEdit {
 	void set_auto_indent_enabled(bool p_enabled);
 	bool is_auto_indent_enabled() const;
 
+	void set_tab_always_indents(const bool p_tab_always_indents);
+	bool is_tab_always_indents() const;
```

Comment:
```suggestion
	bool is_tab_always_indents_enabled() const;
```
Unsure about the wording but the current wording is incomplete